### DISCUSSION TITLE
CHANGE DEFAULT AUGMENTATION VALUES VIA YOLO PACKAGE

### DISCUSSION THREAD

**Discussion by edanurkamaci** [ultralytics/ultralytics#2427] [Q&A]
Hi I wanted to augment my dataset using augment class which has albumentation operations with yolo package. I did it but I do not know how to integrate it to my model created by YOLO(..) function. Can you help me with that. I do not wanna clone github code, I am just trying to figure out to how to deal with it by using package

**Comment by glenn-jocher** [comment]
@edanurkamaci hello! To integrate custom augmentations using the Albumentations library with your YOLOv8 model created by the `YOLO()` function, you'll need to adjust the augmentation settings in your training configuration file. This file is typically a YAML file where you specify various training parameters, including augmentation options.

Once you've defined your custom augmentations in the Albumentations format, you can reference them in your configuration file under the appropriate section. Make sure that the augmentations you define are compatible with the YOLOv8 model's expectations.

After updating your configuration file, you can pass it to the `YOLO()` function when initializing your model for training. The model will then apply your specified augmentations during the training process.

For more detailed instructions on how to customize your training configuration, please refer to the Ultralytics Docs, specifically the section on training. Keep in mind that while customizing augmentations, it's important to ensure they are suitable for the object detection task and that they do not negatively impact the model's ability to learn from your dataset.

Happy training! 😊🚀
